source-map-uploader-plugin.js

```diff
@@ -133,12 +138,6 @@ class BugsnagSourceMapUploaderPlugin {
   }
 
   bugsnagCliUploadOpts (sm) {
-    // Validate required fields
-    if (!this.apiKey) {
-      console.error('Error: API key is required but was not provided.')
-      return null
-    }
```

Comment:
do we not need this any more?

Reply:
Correct. As this was handled by `this.validate()` the API Key should never not be set when we get here.